[2024-12-05T09:31:02.992+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-05T09:31:03.011+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test scheduled__2024-12-04T07:00:00+00:00 [queued]>
[2024-12-05T09:31:03.022+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test scheduled__2024-12-04T07:00:00+00:00 [queued]>
[2024-12-05T09:31:03.023+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-05T09:31:03.044+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_test> on 2024-12-04 07:00:00+00:00
[2024-12-05T09:31:03.185+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4973) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T09:31:03.191+0000] {standard_task_runner.py:63} INFO - Started process 4975 to run task
[2024-12-05T09:31:03.196+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'BuildELT_dbt', 'dbt_test', 'scheduled__2024-12-04T07:00:00+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/build_elt_with_dbt.py', '--cfg-path', '/tmp/tmpi6jiuwum']
[2024-12-05T09:31:03.206+0000] {standard_task_runner.py:91} INFO - Job 111: Subtask dbt_test
[2024-12-05T09:31:03.271+0000] {task_command.py:426} INFO - Running <TaskInstance: BuildELT_dbt.dbt_test scheduled__2024-12-04T07:00:00+00:00 [running]> on host 2adeb2e19af7
[2024-12-05T09:31:03.394+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BuildELT_dbt' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2024-12-04T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-04T07:00:00+00:00'
[2024-12-05T09:31:03.406+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-05T09:31:03.408+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-05T09:31:03.415+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /opt/***/final_dbt --project-dir /opt/***/final_dbt']
[2024-12-05T09:31:03.458+0000] {subprocess.py:86} INFO - Output:
[2024-12-05T10:03:22.480+0000] {subprocess.py:93} INFO - [0m10:03:22  Running with dbt=1.8.7
[2024-12-05T10:03:23.627+0000] {subprocess.py:93} INFO - [0m10:03:23  Registered adapter: snowflake=1.8.4
[2024-12-05T10:03:23.958+0000] {subprocess.py:93} INFO - [0m10:03:23  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-12-05T10:03:23.960+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2024-12-05T10:03:23.961+0000] {subprocess.py:93} INFO - - models.final_dbt.example
[2024-12-05T10:03:24.191+0000] {subprocess.py:93} INFO - [0m10:03:24  Found 2 snapshots, 3 models, 2 sources, 459 macros
[2024-12-05T10:03:24.199+0000] {subprocess.py:93} INFO - [0m10:03:24
[2024-12-05T10:03:24.202+0000] {subprocess.py:93} INFO - [0m10:03:24  Nothing to do. Try checking your model configs and model specification args
[2024-12-05T10:03:25.249+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-05T10:03:25.252+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-05T10:03:25.285+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=BuildELT_dbt, task_id=dbt_test, run_id=scheduled__2024-12-04T07:00:00+00:00, execution_date=20241204T070000, start_date=20241205T093103, end_date=20241205T100325
[2024-12-05T10:03:25.357+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-05T10:03:25.385+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-05T10:03:25.388+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
